packages/zambdas/src/scripts/deploy-zambdas.ts

```diff
@@ -53,10 +53,7 @@ Object.entries(ottehrSpec.zambdas).forEach(([_key, spec]) => {
     };
   }
 
-  if (
-    process.env.environment !== 'demo' &&
-    (spec.name === 'SEND-MESSAGE-CRON' || spec.name === 'NOTIFICATIONS-UPDATER')
-  ) {
+  if (!['staging', 'demo'].includes(process.env.environment as string) && spec.name === 'SEND-MESSAGE-CRON') {
```

Comment:
can you please make this list of envs a variable and include it in the log in the next line?

Reply:
701ff03